### PR TITLE
[Mo 3874] - iOS Swift Example App

### PR DESCRIPTION
**Pull request by arnaumartifont**

### CODE REVIEW

File: Swift/ChartboostExample copy-Info.plist [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (daniel-barros) [inline Swift/ChartboostExample copy-Info.plist:1]: I believe this file is not needed. There's already an Info.plist inside the project. Can we remove this?

Comment (arnaumartifont) [reply Swift/ChartboostExample copy-Info.plist:1]: Yes, I'll switch the pods target to use the same Info.plist, thanks for that!

Comment (daniel-barros) [reply Swift/ChartboostExample copy-Info.plist:1]: Actually, I see now that this is the Info.plist required for the Pods target.
I think we should rename it to "ChartboostExample-Pods-Info.plist", same as the Obj-C one, put it in the same folder as the other Info.plist (I would rename that one to "ChartboostExample-Info.plist", same as the Obj-C one), and fix the link to it in the project's Build Settings (Packaging > Info.plist file).
Also make sure that both plist files have locations "relative to group" and not "absolute path)

Comment (daniel-barros) [reply Swift/ChartboostExample copy-Info.plist:1]: Nvm what I just said, sharing the same "Info.plist" makes a lot of sense

File: Swift/ChartboostExample/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        interstitial.show(from: self)
+    }
+
+    @IBAction func cacheRewarded(_ sender: Any) {
+        rewarded.cache()
+    }
+
+    @IBAction func showRewarded(_ sender: Any) {
+        // We can let showFromViewController: fail for not-cached ads as we do in showInterstitial:, or preemtively check against the isCached property before calling it:
+        if rewarded.isCached == true {
```
Comment (daniel-barros) [inline Swift/ChartboostExample/ViewController.swift:42]: I think `if rewarded.isCached {` is better. This was probably necessary when rewarded was an optional but not anymore.

**Pull request merged**